extensions/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/CamelBeanBuildItem.java

```diff
@@ -20,18 +20,22 @@
 
 import io.quarkus.builder.item.MultiBuildItem;
 
-public final class CamelRegistryBuildItem extends MultiBuildItem {
+/**
+ * A MultiBuildItem holding beans to add to CamelRegistry during static initialization phase.
+ * Can be produced only by methods that do not depend on CamelRuntime (because otherwise there is a circular dependency).
+ */
```

Comment:
The text is perfect, but having MultiBuildItem CamelRegistry and CamelRuntime as links (or at least FQ names) would be nice.

Reply:
done